src/components/GetHttpURL/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Excuse me, sir, what is the functionality of this component?

Reply:
I was / I am yet. xd trying to get the document.location.href to compare it with that href with the link that the buttons has :v and also I know if that could works with the back button, but I'm still investigating the snapshot testing and those situations